hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/keys/GetKeyHandler.java

```diff
@@ -65,8 +65,10 @@ public Void call() throws Exception {
 
     OzoneAddress address = new OzoneAddress(uri);
     address.ensureKeyAddress();
-    try (OzoneClient client =
-             address.createClient(createOzoneConfiguration())) {
+
+    OzoneConfiguration conf = createOzoneConfiguration();
+
+    try (OzoneClient client = address.createClient(conf)) {
 
```

Comment:
@adoroszlai Can we avoid creating variable `conf` as it is only needed to createClient and not used later?
We can stick with original code and you could make any checkstyle/formatting changes without creating a local variable.

Reply:
It _is_ used to get chunk size.

Reply:
Ah! I see it now. I didn't have my coffee yet 😆